### ISSUE TITLE
How can I capture the output log? 

### ISSUE THREAD

**Issue by leoniedu** [dirkschumacher/rcbc#70]
Sink (message or output) does not seem to work.

**Comment by jeffreyhanson** [comment]
Yeah, I don't know how to do this in an interactive session sorry. This might be possible in a non-interactive session if you have it save a log file? E.g., try running something like this from the command line:

```
R CMD BATCH --no-restore --no-save myscript.R
```

This should create a myscript.Rout file that has a log of the R code and outputs.
